pages/onboarding.js

```diff
@@ -0,0 +1,114 @@
+import { useState, useEffect } from 'react'
+import BoxLoading from '../components/BoxLoading/index'
+import Head from 'next/head'
+import Footer from '../components/Footer'
+import Navbar from '../components/Navbar'
+
+const Onboarding = () => {
+  const [isLoading, setIsLoading] = useState(true)
+  const [account, setAccount] = useState(null)
+  const [balance, setBalance] = useState('')
+  const [selectedVerifier, setSelectedVerifier] = useState('google')
+  const [verifierId, setVerifierId] = useState(null)
+  const [buildEnv, setBuildEnv] = useState('development')
+  const [web3Obj, setWeb3Obj] = useState(null)
+
+  useEffect(() => {
+    setIsLoading(true)
+    async function loadTorus() {
+      const { default: web3Obj } = await import('../lib/helper')
+      setWeb3Obj(web3Obj)
+    }
+    loadTorus()
+      .then(() => {
+        setIsLoading(false)
+      })
+      .catch(e => console.log(e))
+  }, [])
+
+  useEffect(() => {
+    const isTorus = sessionStorage.getItem('pageUsingTorus')
+    if (isTorus && web3Obj) {
+      web3Obj.initialize(isTorus).then(() => {
+        userInfo()
+      })
+    }
+    setIsLoading(false)
```

Comment:
shouldn't we setIsLoading(false) after userInfo() call completes? 
maybe userInfo should be async and we call userInfo() after awaiting?